dao/FashionKnex.js

```diff
@@ -80,29 +84,74 @@ class FashionKnex {
       })
   }
 
+  /**
+  * Find a Entity by `id` in the database
+  *
+  * @param id {String} - Id to search for
+  * @param options {Object}
+  *   options.select {String|Array} - Properties to select in the query
+  */
   async findById (id, {select = '*'} = {}) {
     const tableName = this._tableName
 
-    let wrapped
     let result
 
+    if (typeof select === 'string' || typeof select === 'undefined') {
+      result = this.select(select)
+    } else if (Array.isArray(select)) {
+      result = this.select(...select)
+    } else {
+      throw new Error(`Invalid select value "${select}". The select option ` +
+        `should either be a string or an array.`)
+    }
+
     try {
-      result = await this._knex
-        .select(select)
-        .from(tableName)
+      result = await result
         .where('id', id)
         .limit(1)
     } catch (err) {
       this._logger.error(`Error fetching id "${id}" from  table "${tableName}"`, err)
       throw err
     }
 
-    let errors = []
+    return this.wrap(result[0])
+  }
 
-    wrapped = this._modelType.wrap(result[0], errors)
+  /**
+  * Select query builder that selects from our current table. Uses `arguments`
+  * to determine what is selected.
```

Comment:
Might be better to just link the select docs from knex here since this is just a thin passthrough - "Uses `arguments` to determine what is selected" isn't very informative.